src/python_testing/TC_DeviceConformance.py

```diff
@@ -76,17 +76,24 @@ def record_warning(location, problem):
 
         success = True
         allow_provisional = self.user_params.get("allow_provisional", False)
+        # TODO: automate this once https://github.com/csa-data-model/projects/issues/454 is done.
+        provisional_cluster_ids = [Clusters.ContentControl.id, Clusters.ScenesManagement.id, Clusters.BallastConfiguration.id,
```

Comment:
you will need to add Dish washer Alarm cluster to this list

Reply:
Cluster itself is not provisional, it just has provisional features. This will be captured by the conformance tests for the elements.

Reply:
it did not pass the rule of 3 platforms in SVE, hence this was requested to be marked as provisional and all cert tests are removed.

Reply:
The cluster is not marked as provisional in the spec? https://github.com/CHIP-Specifications/connectedhomeip-spec/blob/master/src/app_clusters/DishwasherAlarm.adoc

Reply:
Discussed this on slack. Cluster is not provisional - there is one feature that can be certified. The provisional features will be caught by the element-level tests. 

Going to resolve this conversation now, since there is nothing to do here.